composer.json

```diff
@@ -70,5 +70,11 @@
             "phpstan/extension-installer": true
         },
         "vendor-dir": "src/vendor"
+    },
+    "scripts": {
+        "post-install-cmd": [
+            "npm install",
```

Comment:
Can `pnpm` also be called by entering `npm`? 
If it can I'm fine with this, otherwise I'm not as `pnpm` is an alternative to `npm` that others may have installed on their system instead of `npm`

Reply:
No you can't

Reply:
removed.